src/SidebarProvider.tsx

```diff
@@ -15,20 +15,16 @@ export const SidebarContext = React.createContext<ISidebarContextInterface | nul
 
 export function SidebarProvider({
   children,
-}: ISidebarProviderProps): JSX.Element {
+}: ISidebarProviderProps) {
```

Comment:
Are you removing the type as a preference? Does this still pass tslint? (we don't have pipeline to fail the PR set up atm)

Reply:
Oh yeah I forgot to mention this. I'd recommend removing the `JSX.Element` and infer from return type. Right now you are upcasting the context as just `JSX.Element`, but it's actually much more than just an element so you will be losing a lot of typing power 😀